draft-dcook-ppm-dap-interop-test-design.md

```diff
@@ -92,7 +92,8 @@ Each of these test APIs should return a status code of 200 OK if the command was
 In multiple APIs defined below, the test runner will send the name of a VDAF, along with the parameters necessary to fully specify the VDAF. These will be stored in a nested object, with the following attributes (new `type` values and new keys will be added as new VDAFs are defined).
 
 |Key|Value|
-|`type`|One of `"Prio3Aes128Count"`, `"Prio3Aes128Sum"`, or `"Prio3Aes128Histogram"`|
+|`type`|One of `"Prio3Aes128Count"`, `"Prio3Aes128CountVec"`, `"Prio3Aes128Sum"`, or `"Prio3Aes128Histogram"`|
+|`length` (only present if `type` is `"Prio3Aes128CountVec`")|The length of the vector, used to parameterize the Prio3Aes128CountVec VDAF.|
 |`bits` (only present if `type` is `"Prio3Aes128Sum"`)|The bit width of the integers being summed, (as a number) used to parameterize the Prio3Aes128Sum VDAF.|
 |`buckets` (only present if `type` is `"Prio3Aes128Histogram"`)|An array of histogram bucket boundaries, (encoded in base 10 as strings) used to parameterize the Prio3Aes128Histogram VDAF.|
 {: title="VDAF JSON object structure" #vdaf-object}
```

Comment:
Ideally we could define this without discussing specific VDAFs, but I haven't worked out a way to achieve that.

Reply:
I think that would have to start with VDAF defining a portable means for communicating a VDAF and its parameters.

Reply:
I think maybe VDAF has to anyway. It implicitly does, because the [test vectors](https://www.ietf.org/archive/id/draft-irtf-cfrg-vdaf-03.html#name-test-vectors) it publishes include ad-hoc encodings of the VDAF parameters. `Prio3Aes128Sum` has `bits: 8` and `Prio3Aes128Histogram` has `buckets: [1, 10, 100]`.